### DISCUSSION TITLE
New fingerbot zigbee

### DISCUSSION THREAD

**Discussion by Grazulex** [Koenkk/zigbee2mqtt#17847] [Devices]
Hello,

I have find a zigbee fingerbot 
https://www.meshhub.com/manuals/adfbz301-zx-en

When a try the pairing it is recognize lika a tuya button Switch and only one action is working...

What Can info to help you to put this New device in the liste and make it working fully?

**Comment by stijn001** [comment]
indeed, it only acts as a push button. No parameter settings in Z2M. Would be nice to have on/ off,  and set duration of push.

**Comment by kristofejro** [comment]
Probably someone with tuya gateway needs to capture traffic (some debug log?) and provide that info to @Koenkk 
Regarding device recognition @cloudbr34k84 already opened [issue ](https://github.com/Koenkk/zigbee2mqtt/issues/17841) and it should be recognized in latest release.
